KsApi/models/Project.swift

```diff
@@ -4,6 +4,7 @@ import Prelude
 import Runes
 
 public struct Project {
+  public var availableCardTypes: [String]?
```

Comment:
This is optional because the project JSON object we get from the `/discover` endpoint doesn't have this field (it's only a partial project), and we use the same `Project` model to decode that payload.

Reply:
I suppose we could also consider coalescing an empty array if we wanted this to be non-optional 🤔 but I guess having `nil` here is more explicit about _not_ having the information rather than there are no available card types.

Reply:
Yeah I'd rather keep it `nil` to make it clear that the info is not available.